src/renderers/dom/stack/client/wrappers/ReactDOMInput.js

```diff
@@ -174,14 +171,17 @@ var ReactDOMInput = {
     var node = ReactDOMComponentTree.getNodeFromInstance(inst);
     var value = props.value;
     if (value != null) {
-
-      // Cast `value` to a string to ensure the value is set correctly. While
-      // browsers typically do this as necessary, jsdom doesn't.
-      var newValue = '' + value;
-
-      // To avoid side effects (such as losing text selection), only set value if changed
-      if (newValue !== node.value) {
-        node.value = newValue;
+      if (value === 0) {
```

Comment:
Huh... was there a recent fix for inputs in Chrome? I can assign `node.value = "0.0000"` and it totally just works... I don't believe that was ever the case before.